### ISSUE TITLE
Database Error max key length on fresh install 

### ISSUE THREAD

**Issue by Maybach91** [InvoicePlane/InvoicePlane#886]
I got the following error for a fresh beta install on php@8.1:

```
A Database Error Occurred
Error Number: 1071

Specified key was too long; max key length is 1000 bytes

CREATE TABLE `ip_login_log` ( `login_name` varchar(255), `log_count` int DEFAULT 0, `log_create_timestamp` datetime, PRIMARY KEY (login_name) );

Filename: modules/setup/models/Mdl_setup.php

Line Number: 51
```
at this step: `index.php/setup/upgrade_tables`

mysql `5.7.37`

**Comment by naui95** [comment]
Hi @Maybach91, interesting issue, thanks for reporting it. On (i) what `engine` and (ii) what `collation` is your myql database using? Do I understand correctly that this was not an upgrade but just new install?

**Comment by naui95** [comment]
Hi @Maybach91, well actually MyISAM should work, since InvoicePlane is set to use that engine. If you get the chance can I kindly ask you to check if setting `VARCHAR(100)` in this file [application/modules/setup/sql/035_1.5.12.sql](https://github.com/InvoicePlane/InvoicePlane/blob/56a87244cf8bc68e4f14d8dd8f3af3fa9625f191/application/modules/setup/sql/035_1.5.12.sql) in your local environment solves the issue.

**Comment by Maybach91** [comment]
Thanks @naui95 , this solved my installation! Now i have no user created, but thats another issue. :D

**Comment by naui95** [comment]
@Maybach91 perfect. For the moment I'll leave it open so we remember to fix it before the release of v1.6.0.

**Comment by nielsdrost7** [comment]
fixed in the upcoming 1.6 release. If you do the usual by updating the database (one of the steps in the documentation) the field length will be corrected.
